daemon/lokinet-vpn.cpp

```diff
@@ -1,25 +1,19 @@
-#include <oxenmq/oxenmq.h>
-#include <nlohmann/json.hpp>
+#include <CLI/CLI.hpp>
 #include <fmt/core.h>
+#include <nlohmann/json.hpp>
+#include <oxenmq/oxenmq.h>
+
 #include <future>
-#include <vector>
-#include <array>
-#include <llarp/net/net.hpp>
 #include <string_view>
-
-#include <CLI/App.hpp>
-#include <CLI/Formatter.hpp>
-#include <CLI/Config.hpp>
-#include "oxenmq/address.h"
+#include <vector>
 
 #ifdef _WIN32
 // add the unholy windows headers for iphlpapi
-#include <winsock2.h>
```

Comment:
`winsock2.h` is a special holy water include that changes other includes based on when it gets included, and so generally *has* to be before other Windows-specific includes.  Yay Windows.

Reply:
IWYU